app/controllers/users/sessions_controller.rb

```diff
@@ -3,13 +3,17 @@ class Users::SessionsController < Devise::SessionsController
   private
 
     def after_sign_in_path_for(resource)
-      if !verifying_via_email? && resource.show_welcome_screen?
+      if !verifying_via_email? && resource.show_welcome_screen? && !override?
         welcome_path
       else
         super
       end
     end
 
+    def override?
+      Setting["feature.user.override_verification"] == 'active'
```

Comment:
How about something like this for the name of the setting?
`Setting["feature.user.skip_verification"] == true`

The true is also important, all other features are either true or nil